packages/suite/src/components/firmware/Initial.tsx

```diff
@@ -171,19 +171,19 @@ const BottomBar = () => {
         return null;
     }
 
-    if (['outdated', 'required'].includes(device.firmware)) {
-        return (
-            <>
-                <ContinueButton onClick={() => setStatus('check-seed')} />
-                <HowLong>
-                    <Icon size={12} icon="CLOCK" />
-                    <Translation id="TR_TAKES_N_MINUTES" values={{ n: '5' }} />
-                </HowLong>
-            </>
-        );
-    }
+    // if (['outdated', 'required'].includes(device.firmware)) {
```

Comment:
Let's just remove this code altogether. .)

Reply:
I'll add handy comment, this was removed in intermediary fw branch to allow reinstalling fw on devices that already have fw installed. I am not sure if this was some official request or not, so let's keep it fo future generations to make their life easier 😁